docs/v0.7/en/greptimecloud/usage-&-billing/overview.md

```diff
@@ -0,0 +1,9 @@
+# Overview
+
+These documents will help you understand the usage and billing of Greptime Cloud.
+
+- [WCU and RCU](wcu-rcu.md)
+- [Hobby Plan](hobby.md)
+- [Serverless Plan](serverless.md)
+- [Dedicated Plan](dedicated.md)
+- [BYOC](byoc.md)
```

Comment:
I'm a bit hesitant if we should use "Bring Your Own Cloud" here. It's long and the concept is well known.

Also if we should recommend BYOC over dedicated so we move it one level above.

Reply:
The [pricing page]((https://greptime.com/pricing)) using BYOC as the plan name, so the documentation uses the same name to guide users from webiste.